src/store/tokenStore.js

```diff
@@ -53,6 +67,16 @@ const actions = {
 	updateToken(context, newToken) {
 		context.commit('updateToken', newToken)
 	},
+
+	/**
+	 * Updates the file ID for the current token
+	 *
+	 * @param {object} context default store context
+	 * @param {int} newFileId The file ID of the active conversation
+	 */
+	updateFileIdForToken(context, newFileId) {
```

Comment:
Should that be done in 1 update method, so we don't accidentally do a mixed state with 1 file id and 1 token of another file in the meantime?

Reply:
I thought about it but as they were updated synchronously and in pairs that should not be an issue. Nevertheless, the other way sounds cleaner, so I will do that.